features/edit_destroy.feature

```diff
@@ -0,0 +1,106 @@
+@wip
+Feature: engagements and iterations have edit and destroy buttons
+  As a staff of the course
+  So that I can manage the engagements and iterations
+  I want to add edit and destroy buttons to engagement page and iteration page
+
+Background: Logged in
+  
+  Given the following apps exist:
+    | id | name  | description | org_id | status  |
+    | 1  | app1  | test        | 1      | pending |
+    | 2  | app2  | test        | 1      | pending |
+    | 3  | app3  | test        | 1      | pending |
+
+  And the following orgs exist:
+    | id | name | contact_id |
+    | 1  | org1 | 1          |
+    | 2  | org2 | 1          |
+    | 3  | org3 | 1          |
+
+  And the following users exist:
+    | id | name  | github_uid      | email          |
+    | 1  | user1 | esaas_developer | test@user.com  |
+    | 2  | user2 |                 | test1@user.com |
+    | 3  | user3 |                 | test2@user.com |
+    | 4  | user4 |                 | test3@user.com |
+    | 5  | user5 |                 | test@user.com |
+    | 6  | user6 |                 | test@user.com |
+
+  And I'm logged in on the orgs page
+  And I am logged in as a "staff"
+  And I follow "Apps"
+  And I follow "app1"
+  
+Scenario: Can create and edit multiple engagements
+  #Story ID: #152689950
+  And I create a new engagement for "app1"
+  And I fill in the engagement fields as follows:
+       | field                  | value      |
+       | Team number            | Team1      |
+       | Student names          | Student1   | 
+  And I select "user1 user2 user3" as Team members
+  And I press "Save"
+  And I create a new engagement for "app1"
+  And I fill in the engagement fields as follows:
+       | field                  | value      |
+       | Team number            | Team2      |
+       | Student names          | Student2   |
+  And I select "user4 user5 user6" as Team members
+  And I press "Save"
+  
+  Then I should see "Team1" has button "Edit"
+  And I should see "Team2" has button "Edit"
+
+Scenario: Can create and destroy multiple engagements
+  #Story ID: #152689950
+  And I create a new engagement for "app1"
+  And I fill in the engagement fields as follows:
```

Comment:
You might want to refactor adding the engagements to the background section since we're really just trying to test the appearance and functionality of the buttons, not creating engagements

Reply:
Thanks Joe for the suggestion. I agree and just pushed a refactored updated version.